### PR TITLE
fix: Lightnet docs

### PR DESCRIPTION
**Pull request by barriebyron**
This PR closes https://github.com/o1-labs/docs2/issues/821 with fixes to the Lightnet docs

Thank you @Sventimir  for the detailed suggestions for updates

### CODE REVIEW

File: docs/zkapps/testing-zkapps-lightnet.mdx [hunk 1]
```diff
@@ -74,12 +77,12 @@ This command performs the following operations that successfully serve the major
 
 - Pulls the latest Docker image for your environment from the [Docker Hub](https://hub.docker.com/r/o1labs/mina-local-network) repository
 - Prepares the file system
-- Downloads and updates the lightweight Mina Explorer
 - Forms the network with a single node
 - Configures the network to be fast
 - Disables the blockchain SNARK proofs
 - Uses the `o1js-main` branch of the Mina repository
-- Starts the Mina archive process and the archive node API
+- Starts the Mina archive process and the [archive-node-API](https://github.com/o1-labs/Archive-Node-API) application.
```
Comment (barriebyron) [inline docs/zkapps/testing-zkapps-lightnet.mdx:84]:
```suggestion
- Starts the Mina archive process and the [Archive-Node-API](https://github.com/o1-labs/Archive-Node-API) application.
```

File: docs/zkapps/testing-zkapps-lightnet.mdx [hunk 2]
```diff
@@ -144,12 +147,12 @@ To monitor transactions, launch the lightweight Mina Explorer:
...
 ```
 
-By default, the latest version is launched. 
+This command downloads and updates a lightweight Mina explorer. By default, the latest version is downloaded (if needed) and launched. 
 
 To list versions, their published dates, and show the version in use:
 
 ```sh
-zk lightnet explorer list true
+zk lightnet explorer list
```
Comment (shimkiv) [inline docs/zkapps/testing-zkapps-lightnet.mdx:155]:
```suggestion
zk lightnet explorer --list
```

File: docs/zkapps/testing-zkapps-lightnet.mdx [hunk 3]
```diff
@@ -144,12 +147,12 @@ To monitor transactions, launch the lightweight Mina Explorer:
 zk lightnet explorer
 ```
 
-By default, the latest version is launched. 
+This command downloads and updates a lightweight Mina explorer. By default, the latest version is downloaded (if needed) and launched.
```
Comment (shimkiv) [inline docs/zkapps/testing-zkapps-lightnet.mdx:150]:
```suggestion
By default, this command downloads (if needed) and launches the latest version of [lightweight Mina explorer](https://github.com/o1-labs/mina-lightweight-explorer). 
```

**Review by shimkiv — CHANGES_REQUESTED** [review]

File: docs/zkapps/testing-zkapps-lightnet.mdx [hunk 4]
```diff
@@ -34,6 +34,7 @@ Use of Lightnet is appropriate for local development and testing only. It is not
 
 ## Prerequisites
 
+1. [Install zkApp CLI](/zkapps/install-zkapp-cli). 
 
 1. [Write tests for your smart contract](/zkapps/testing-zkapps-locally#writing-tests-for-your-smart-contract).
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply docs/zkapps/testing-zkapps-lightnet.mdx:39]: Yes, it's fine https://github.com/o1-labs/docs2/wiki/Markdown-Syntax#numbered-steps 
Using 1. for each step simplifies maintenance
Here's the handy preview link https://docs2-git-iterate-lightnet-minadocs.vercel.app/zkapps/testing-zkapps-lightnet#prerequisites

File: docs/zkapps/testing-zkapps-lightnet.mdx [hunk 5]
```diff
@@ -58,6 +59,8 @@ https://docs.docker.com/engine/install/).
 1. Develop, iterate, and [troubleshoot](#troubleshoot-transactions)
 1. [Stop the local network](#stop-the-local-network)
 
+The best way experience Lightnet is by using it from the zkApp CLI command line.
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply docs/zkapps/testing-zkapps-lightnet.mdx:62]: oof thankies eagle eye!

**Review by shimkiv — APPROVED** [review]

**Pull request merged**